tests/rest/client/v2_alpha/test_shared_rooms.py

```diff
@@ -41,13 +42,12 @@ def prepare(self, reactor, clock, hs):
         self.handler = hs.get_user_directory_handler()
 
     def _get_shared_rooms(self, token, other_user):
-        request, channel = self.make_request(
+        channel = self.make_request(
             "GET",
             "/_matrix/client/unstable/uk.half-shot.msc2666/user/shared_rooms/%s"
             % other_user,
             access_token=token,
         )
-        return request, channel
```

Comment:
I think you still want to return the `channel` here?

Reply:
so I do. thank you.